src/textual/events.py

```diff
@@ -563,22 +571,24 @@ class Blur(Event, bubble=False):
 class AppFocus(Event, bubble=False):
     """Sent when the app has focus.
 
-    Only available when running within a terminal that supports `FocusIn`,
-    or when running via textual-web.
-
     - [ ] Bubbles
     - [ ] Verbose
+
+    Note:
+        Only available when running within a terminal that supports
+        `FocusIn`, or when running via textual-web.
     """
 
 
 class AppBlur(Event, bubble=False):
-    """Sent when the app loses focus.
-
-    Only available when running within a terminal that supports `FocusOut`,
-    or when running via textual-web.
+    """Sent when the app gains focus.
```

Comment:
I don't envy you working through all these! Hope you don't mind me sticking my nose in as usual, but spotted this earlier.

```suggestion
    """Sent when the app loses focus.
```

Reply:
Good catch! I have *no* idea what happened there; can't for the life of me figure out what I was thinking.